x-pack/plugins/observability_solution/apm/public/components/app/service_tab_empty_state/constants.ts

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export type EmptyStateKey =
+  | 'serviceOverview'
+  | 'serviceDependencies'
+  | 'infraOverview'
+  | 'serviceMap'
+  | 'transactionOverview'
+  | 'metrics'
+  | 'errorGroupOverview';
+
+interface EmptyStateContent {
+  title: string;
+  content: string;
+  imgName?: string;
+}
+
+export const emptyStateDefinitions: Record<EmptyStateKey, EmptyStateContent> = {
```

Comment:
nit: In some title we use services and in other application not sure if that was on purpose 🤔

Reply:
@roshan-elastic